cf/read_write/netcdf/netcdfwrite.py

```diff
@@ -175,9 +254,10 @@ def _write_dimension_coordinate(
 
             ncdim: `str` or `None`
                 The name of the netCDF dimension for this dimension
-                coordinate construct, including any groups structure. Note
-                that the group structure may be different to the
-                corodinate variable, and the basename.
+                coordinate construct, including any groups
+                structure. Note that the group structure may be
+                different to the corodinate variable, and the
```

Comment:
```suggestion
                different to the coordinate variable, and the
```
(though the typo was from before, might as well fix it here.)